app/views/auth_source_ldaps/_form.html.erb

```diff
@@ -27,7 +27,7 @@
                     { :label => _('Server type'), :onchange => 'change_ldap_server_type()' } %>
     </div>
     <div class="tab-pane" id="account">
-      <%= text_f f, :account, :help_inline =>_("Use this account to authenticate, <i>optional</i>").html_safe, :label_help => account_help_data[@auth_source_ldap.server_type], :data => { :help => account_help_data } %>
+      <%= text_f f, :account, :help_inline =>_("Use this account to authenticate, <i>optional</i>").html_safe, :label_help => account_help_data[@auth_source_ldap.server_type], :data => { :help => account_help_data }, :autocomplete => 'new-account' %>
```

Comment:
I can't get to reproduce the issue, I do have my credentials saved in Firefox yet they're not autocomplete. Anyway this looks like it would do it, but does it autocomplete for `new-account`? 

What do you think of setting `:html => {:autocomplete => "off"}` at the `form_for` so that it's disabled in the entire form?

Reply:
This happens to me in chrome as well. Chrome ignores `:autocomplete => "off"` sadly. however when we use 'new-account' chrome thinks this is a new account name and doesn't try to autofill it. we use a similar approach for passwords.